zipline/pipeline/loaders/blaze/_core.pyx

```diff
@@ -524,6 +524,13 @@ cdef array_for_column(object dtype,
         raise TypeError('unknown column dtype: %r' % input_array.dtype)
 
 
+cpdef str getname(object column):
+    try:
+        return column.metadata['blaze_column_name']
```

Comment:
We should have a test that writes the `blaze_column_name` so that this code path is exercised. 

Also will `metatadata` always exist? If not, we may want to add catching an `AttributeError`